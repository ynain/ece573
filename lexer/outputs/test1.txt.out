Token Type: INTLITERAL
Value: 2
Token Type: OPERATOR
Value: +
Token Type: INTLITERAL
Value: 2
Token Type: OPERATOR
Value: =
Token Type: INTLITERAL
Value: 4
Token Type: OPERATOR
Value: -
Token Type: INTLITERAL
Value: 1
Token Type: OPERATOR
Value: =
Token Type: INTLITERAL
Value: 3
Token Type: IDENTIFER
Value: quick
Token Type: IDENTIFER
Value: maths
