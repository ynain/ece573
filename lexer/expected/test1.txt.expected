Token Type: INTLITERAL
Value: 2
Token Type: OPERATOR
Value: +
Token Type: INTLITERAL
Value: 2
Token Type: OPERATOR
Value: =
Token Type: INTLITERAL
Value: 4
Token Type: OPERATOR
Value: -
Token Type: INTLITERAL
Value: 1
Token Type: OPERATOR
Value: =
Token Type: INTLITERAL
Value: 3
Token Type: IDENTIFIER
Value: quick
Token Type: IDENTIFIER
Value: maths
